packages/next/src/client/components/react-dev-overlay/_experimental/internal/hooks/use-keyboard-shortcut.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect } from 'react'
+
+export const MODIFIERS = {
+  CTRL_CMD: 'CTRL_CMD',
+  ALT: 'ALT',
+  SHIFT: 'SHIFT',
+} as const
+
+type KeyboardShortcutProps = {
+  key: string
+  cb: () => void
```

Comment:
if it's a key let's give a full name like `callback`
```suggestion
  callback: () => void
```